optuna/integration/cma.py

```diff
@@ -519,6 +474,7 @@ def _to_optuna_params(
                 return float(min(max(v, dist.low), dist.high))
             else:
                 return float(cma_param_value)
+
         if isinstance(dist, IntDistribution):
```

Comment:
I know you leave the original implementation. But I think we can merge these branches. 🙋

Reply:
You mean the ones in `_to_optuna_params`? I'm not sure, they seem to differ quite a bit between `FloatDistribution` and `IntDistribution`.

Reply:
Ah, sorry for the ambiguous comment. No I don't. I mean I think we can merge two if blocks look like:

### Before

```python
if isinstance(dist, FloatDistribution):
    ...
if isinstance(dist, IntDistribution):
    ...
```

### After

```python
if isinstance(dist, FloatDistribution):
    ...
elif isinstance(dist, IntDistribution):
    ...
```

Reply:
Ah, yes, makes sense, thank you for explaining and done in 6abd4fcefc408f020a5252544cc3efae0b3f49ec.